app/assets/javascripts/enroll_camera.js

```diff
@@ -0,0 +1,42 @@
+$(document).ready(function(){
+  
+
+  $('#shutter-button').on('click', function(){
+    take_snapshot()
+    removeShutter()
+    addSubmit()
+  })
+
+  $('div.button').on('click', '#submit-button', function(e) {
+    e.preventDefault()
+    let image = $('#my_camera').children('img').attr('src')
+    $.ajax({
+      type: "POST",
+      url: '/enrollments',
+      data: {image: image},
+      success: function(data){
+
+      }
+    })
+  })
+
+  $('div.button').on('click', '#reset-button', function(e) {
+    e.preventDefault()
+    location.reload()
+  })
+
+  function take_snapshot() {
+    Webcam.snap( function(data_uri) {
+      document.getElementById('my_camera').innerHTML = '<img src="'+data_uri+'"/>';
```

Comment:
here you're using raw JS - any chance you can leverage jQuery to make this cleaner?